src/main/java/roomescape/time/ReservationTimeRepository.java

```diff
@@ -0,0 +1,51 @@
+package roomescape.time;
+
+import java.time.LocalTime;
+import java.util.List;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class ReservationTimeRepository {
+
+    private final JdbcTemplate jdbcTemplate;
+
+    public ReservationTimeRepository(JdbcTemplate jdbcTemplate) {
+        this.jdbcTemplate = jdbcTemplate;
+    }
+
+    public synchronized void insert(final ReservationTime reservationTime) {
+        final String sql = "INSERT INTO reservation_time (start_at) VALUES (?)";
+        jdbcTemplate.update(
+                sql,
+                reservationTime.startAt());
```

Comment:
사소하지만 update 메서드는 한줄로 써도 되지 않을까 싶어요. 👀 
```suggestion
        jdbcTemplate.update(sql, reservationTime.startAt());
```

Reply:
update 메서드를 한줄로 작성하여 좀 더 간결하고 가독성있게 변경하였습니다!